xwiki-platform-core/xwiki-platform-annotations/xwiki-platform-annotation-ui/src/main/resources/AnnotationCode/Translations.xml

```diff
@@ -40,30 +40,30 @@
   <syntaxId>plain/1.0</syntaxId>
   <hidden>true</hidden>
   <content>annotations.title=Annotate
-annotations.menu.loading=Loading annotations settings
+annotations.menu.loading=Loading annotation settings
 annotations.menu.loaderror=Failed: 
-annotations.tab.info.noannotations=No annotations for this page
+annotations.tab.info.noannotations=No annotations on this page
 
 annotations.settings.display=Show annotations 
-annotations.settings.error.wrongsyntax=Annotations are not available for pages in XWiki/1.0 syntax.
+annotations.settings.error.wrongsyntax=Annotations are not available for pages written in XWiki/1.0 syntax.
 annotations.settings.error.notarget=No page specified to get annotations settings for.
 
 annotations.annotated.loading=Loading annotated page
 annotations.annotated.loaderror=Failed: 
 annotations.annotated.loaderror.wrongresponse=Wrongly formatted server response
 annotations.annotated.error.noannotatedelement=Annotations could not be loaded because the content is not available.
-annotations.annotated.error.wrongsyntax=Annotations are not available for pages in XWiki/1.0 syntax.
+annotations.annotated.error.wrongsyntax=Annotations are not available for pages written in XWiki/1.0 syntax.
 
 annotations.action.edit.text=[Edit]
-annotations.action.edit.tooltip=Edit this annotation
+annotations.action.edit.tooltip=Edit annotation
 annotations.action.edit.submit.text=Update
 annotations.action.edit.cancel.text=Cancel
-annotations.action.edit.success=Annotation has been successfully updated.
+annotations.action.edit.success=Annotation updated,
```

Comment:
I just realized my mistake of saying `comma` (`,`) instead of `period` (`.`). Sorry for misleading you, I just mixed up the 2 words in my mind.

However, I do find it interesting on what was you opinion when doing the change. Didn't it feel strange to end a sentence with `,` instead of `.`? :)

Apologies again and please change the `,` to `.` in this and following cases.

Reply:
I initially thought about the period but then I thought it might be intentional.

No problems :)

Reply:
Done.